next-tavla/src/Admin/scenarios/Edit/reducer.tsx

```diff
@@ -12,6 +13,7 @@ export type Action =
     | { type: 'updateTile'; tileIndex: number; tile: TTile }
     | { type: 'swapTiles'; oldIndex: number; newIndex: number }
     | { type: 'toggleLine'; tileId: string; lineId: string }
+    | { type: 'toggleColumn'; tileId: string; column: TColumn; value: boolean }
```

Comment:
Ville kalt denne noe annet enn toggle, eller gjort så den toggler istedet for å sende med en value